src/Execute/DeviceViewProvider.ts

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as cp from 'child_process';
+import * as vscode from 'vscode';
+
+import {globalExecutorArray} from '../Backend/Backend';
+import {Command} from '../Backend/Command';
+import {DeviceSpec, supportedSpecs} from '../Backend/Spec';
+
+import {Device} from './Device';
+import {ExecutionManager} from './ExecutionManager';
+
+type DeviceTreeView = DeviceViewNode|undefined|void;
+
+// This ExecutionManagerMap will be used on Provider.
+interface ExecutionManagerMap {
+  [key: string]: ExecutionManager;
```

Comment:
This key will be a host name for ExecutionManager. 
for example) `local` on local host and IP address for remote connected host